HarryJLeigh.FlashCards/FlashCards/README.md

```diff
@@ -0,0 +1,80 @@
+# Flashcards Project
+
+## PROJECT OVERVIEW
+This is a console-based CRUD application for creating, managing, and studying flashcards, built with **C#**, **Dapper**, **MSSQL**, and **Spectre.Console**. The application allows users to create flashcards, organise them into stacks, and study using an intuitive console-based UI.
+
+The application is designed to improve learning by providing features like:
+- Flashcard creation and organisation.
+- Interactive study sessions with progress tracking.
+- Comprehensive reporting.
+
+**Note**: 

```

Comment:
🟠 **Missing Step**

💡 Remember you need to instruct the user to create the database too if your app doesn't do it on startup.

Reply:
Updated ReadMe with database creation.